src/components/CentralContainer/Sidebar/SidebarTabs/SearchTabPanel/index.tsx

```diff
@@ -38,21 +51,77 @@ import "./index.css";
  *
  * @return
  */
+// eslint-disable-next-line max-lines-per-function
```

Comment:
```suggestion
```

We should avoid disabling linting rules whenever possible. Please see if the refactoring PR at #196 properly addresses this violation.